imageio/imageio-metadata/src/main/java/com/twelvemonkeys/imageio/metadata/tiff/TIFFWriter.java

```diff
@@ -410,7 +420,14 @@ else if (value instanceof long[]) {
 
                         break;
                     }
-
+                case TIFF.TYPE_ASCII:
+                    String[] strings = (String[]) value;
+                    for (String string : strings) {
+                        byte[] bytes = string.getBytes(StandardCharsets.UTF_8);
+                        stream.write(bytes);
+                        stream.write(0);
```

Comment:
Perhaps extract this (and the code below for single string) to a `writeString(stream, string)` method?

Reply:
i've extracted the duplicate code. But i'm not sure what could be changed for a more efficient converting to bytes?

Reply:
I was thinking about perhaps storing the `byte[]` instead of `String/String[]` in the `TIFFEntry`. Instead of converting the Strings to bytes twice, once for the value count, and once for the actual write...

But I don't think this is used enough to warrant that refactoring just now. I'm fine with the current solution.